.circleci/config.yml

```diff
@@ -88,7 +88,6 @@ jobs:
       - *restore_cache
       - *copy_bazel_config
 
-      # TODO(jelbourn): Update this command to run all tests if the Bazel issues have been fixed.
       - run: bazel build src/...
       - run: bazel test src/...
```

Comment:
Note: I don't to build everything that is not related to the tests. Also just running `//...` seems to break our CI build, so we can revisit this later.